---
layout: conf
title: LDNQImmuno
subtitle: March 14th
order: 5
---

<div class="titlebox">
  <h1>
  2nd London Quantitative Immunology Day
  </h1>
  <div class="box">
  <span style="text-align:left">
    Thursday March 14th 2024 <br /> 
    Pears Building, UCL IIT <br />
    Pond Street, London NW32PP <br />
    Tube station: Belsize Park <br />
  </span>
  </div>
  <p style="margin-top:0em;padding:0.5rem">
    A community day for researchers in the quantitative life sciences from across London. <br />
  </p>
  <p style="margin-top:0em;padding-top:0em;padding-bottom:1em;font-size:150%">


<script id="luma-checkout" src="https://embed.lu.ma/checkout-button.js"></script>

<a
  href="https://lu.ma/event/evt-1MnVzjAgp836VoR"
  class="luma-checkout--button"
  data-luma-action="checkout"
  data-luma-event-id="evt-1MnVzjAgp836VoR"
>
  Register now
</a>

  </p>
</div>

<img style="max-width:55%;margin:0.2rem;padding-right:0;margin-right:0" src="../images/ldnday24.png" align="right">
We aim to bring together researchers with an interest in quantitative immunology to create an **opportunity for sharing knowledge and social exchange**. The London Q-Immuno day will be a day of conviviality and scientific enthusiasm with a dynamic and informal atmosphere. Talks from invited speakers will be interleaved with short presentations by young investigators (contributions welcome!). The schedule includes ample breaks for discussions and, for those interested, we propose to conclude the day in a local pub.

Immunology is being transformed by the application of a multitude of quantitative methods. We want to foster discussions between researchers from diverse backgrounds: immunology, evolution, computational biology, evolution, systems biology, bio-informatics, mathematics and the physics of living system. **No matter your background, you are welcome to join us!**

There is no registration fee, but we encourage prior [registration](https://lu.ma/9rvihe65) to help us gauge attendance.

## Schedule

- 09:30-10:00 Meet and Greet  
<details open>
<summary>
10:00-11:00 Session 1, Chair: Andreas Tiffeau-Mayer
</summary>
<details>
<summary>
<a href="https://www.sydney.edu.au/medicine-health/about/our-people/academic-staff/alexandra-sharland.html">Alexandra Sharland</a> (University of Sydney): <br> 
Characterizing alloreactive T cells in Transplantation
</summary>
<p> Directly alloreactive CD8 T cells recognise epitopes formed by an allogeneic MHC I molecule loaded with an endogenous peptide. Using a combination of mass spectrometry and ex-vivo multimer staining, we have recently identified over 40 immunogenic peptides presented by H-2Kb. Profiling the T cell receptor (TCR) repertoire of CD8 T cells responding to a subset of these epitopes under various transplant conditions can inform our understanding of the molecular basis of alloreactivity and the mechanisms underlying transplant tolerance or rejection.
</p>
</details>
<details>
<summary>
Iñigo Ayestaran (Cambridge): <br>
Neoantigen-specific T-cell Receptors as biomarkers of IDH1-mutant Glioma
</summary>
<p> T-cell receptors (TCRs) determine antigen specificity of T-cells, and they collectively form a repertoire that provides information about past and ongoing immune responses to diseases, including cancers. However, whether detection of cancer-specific TCRs can be used for early detection remains unknown. The brain tumour glioma provides an interesting test case for this idea, as >68% of tumours harbour an immunodominant neoantigen generated by the IDH1 R132H hotspot mutation. To identify TCRs specific to this neoantigen, we analysed TCR repertoires from 21 people who received a peptide vaccine designed around the R132H mutation. By considering the expansion dynamics of TCR clones pre- and post- vaccination, we identified ~300 candidate TCRs that we consider likely to be peptide-specific, across a range of HLA backgrounds. The CDR3 sequences of neoantigen-specific TCRs show modest but statistically significant convergence in their sequences suggesting shared antigen specificity. To validate these candidate neoantigen-specific TCRs, we generated TCR repertoires from peripheral blood obtained at diagnosis from 26 glioma patients from an independent cohort containing 11 IDH1 R132H mutant tumours and 15 tumours that are wild-type for IDH1. We develop a score from each one of these repertoires, based on sequence similarity with the candidate peptide-specific TCRs. This score enables the correct stratification of patient repertoires based on their IDH1 mutation status. This study highlights the opportunities and challenges of TCRs as cancer biomarkers in peripheral blood.</p>
</details>
<details>
<summary>
Ann-Marie Baker (ICR): <br>
Mapping clonal evolution and the T-cell repertoire through the colitic bowel
</summary>
<p>Inflammatory bowel disease (IBD) is a chronic relapsing-remitting condition that increases the lifetime risk of developing colorectal cancer by almost two-fold. In patients with IBD, clonal evolution and field cancerisation precede the development of colitis-associated colorectal cancer (CA-CRC), however the extent and spread of pre-cancerous clones and their co-evolution with the immune microenvironment remains incompletely determined. Consequently clinical practice is poorly informed of how best to detect pre-cancerous clones by endoscopy and accurately predict future cancer risk. In this study, we performed a detailed spatially-resolved analysis of the entire colon of two IBD patients, including low pass whole genome sequencing, RNA sequencing and T cell receptor (TCR) sequencing. We quantified the number and size of mutant clones arising across the length of the colitic bowel and determined their relationship with the T cell immune response. Through this we gained a detailed molecular understanding of the evolutionary dynamics of progression to CA-CRC.</p>
</details>
</details>
- 11:00-11:30 Coffee break  
<details open>
<summary>
11:30-12:30 Session 2, Chair: Leo Swadling
</summary>
<details>
<summary>
<a href="https://www.linkedin.com/in/leonard-wossnig/">Leonard Wossnig</a> (Labgenius): <br>
An active learning approach for the discovery of next-generation multispecific antibodies
</summary>
 <p>The emergence of ML-enabled technology platforms that aim to enhance molecule performance have the potential to revolutionize the way we approach drug discovery. LabGenius is pioneering the use of active learning for the discovery of multispecific antibodies, specifically T-cell engagers for solid tumors.
We have developed a lead optimization platform that generates high-quality data from complex assays for machine learning to decipher design-fitness relationships and guide screening efforts to fruitful areas of the design space. We demonstrate our capability by discovering HER2 TCEs up to 400-fold more tumour-selective than a clinical benchmark. This talk will focus on the underlying technology, the deep integration of predictive assays, data generation, data capturing, and data pre-processing and highlight case studies for the successful application of this platform. </p>
</details>
<details>
<summary>
Anna Huhn (Oxford): <br>
A novel particle-based model to analyse bivalent binding of antibodies
</summary>
 <p>Antibodies play an important role in our adaptive immune system by binding to antigenic structures on the surface of pathogens. This binding leads to the neutralisation and destruction of pathogens. Due to their high antigen specificity, they have become essential tools in clinical diagnosis and treatment. Antibodies have two identical binding arms that allow them to achieve high affinity through bivalent binding. This bivalent binding is crucial for the protective function of antibodies, evidenced by the lack of correlation between monovalent antibody-antigen interaction and neutralisation potency. However, we are lacking the methods to characterise antibodies based on their bivalent binding. This is because of the complexity of bivalent binding, involving spatial and stochastic interactions, which necessitate advanced mathematical modelling for quantification. Here, we introduce a new spatial and stochastic particle-based model that accurately captures the physics and chemistry of bivalent binding for the first time. The model includes a new biophysical parameter, which we have termed the "molecular reach", that quantifies the maximum separation distance between antigens that still allows a single antibody to reach them both. After validating the model, we use it to analyse SARS-CoV-2-specific human antibodies. We found that the molecular reach of an antibody is the strongest correlate of SARS-CoV-2 viral neutralisation. Using the bivalent binding parameters, we could directly predict the concentration of antibody required for viral neutralisation. The model enables us to predict how changes in antibody properties and antigen density affect antibody function. This predictive capability is invaluable for optimising antibodies and will help develop new and improved therapeutics and vaccines. </p>
</details>
<details>
<summary>
Kevin Michalewicz (Imperial): <br>
Predicting and explaining antibody binding affinity with ANTIPASTI
</summary>
 <p>Antibodies are key components of the immune system that play a crucial role in targeting antigens and eliciting an effective immune response. A fundamental characteristic of antibodies is their binding affinity to specific targets, which can be increased through exposure to the target and somatic mutations in the process of maturation. This binding affinity is the result of complex physicochemical and structural determinants: our limited understanding of such determinants hinders our ability to rationally design optimised antibodies for research and therapeutic purposes. In silico methods can thus provide a useful tool to predict antibody-antigen binding affinity in conjunction with experimental assays, by helping to focus the mutational and design strategies and, in turn, potentially reducing time and experimental resources. We present ANTIPASTI, a Deep Learning method to predict antibody binding affinity from structural data of antibody-antigen complexes. ANTIPASTI applies CNNs to residue-residue correlation maps derived from Elastic Network representations of antibody-antigen complexes followed by Normal Mode Analysis. In doing so, ANTIPASTI takes into account structural, energetic and global correlation relationships, and thus achieves state-of-the-art predictive accuracy and generalisation power on published experimental data. Furthermore, ANTIPASTI is interpretable, since the model is informative about which residue-residue correlations give rise to increases and decreases of binding affinity. Indeed, our approach can also be used to find the antibody regions that are most relevant for binding affinity depending on the antigen type, revealing that both structural contacts and long-range correlations play an important role in its determination.</p>
</details>
</details>
- 12:30-13:30 Lunch break
<details open>
<summary>
13:30-14:15 Whiteboard talks and posters
</summary>
<details>
<summary>
Juan Luis Melero (Omniscope): <br>
Spatio-temporal tracking of therapy-induced T cell immunity against pediatric rhabdoid tumors - A success case report
</summary>
<p>Malignant rhabdoid tumors (RT) are rare childhood tumors that initiate during the embryo development and manifest in the kidney, soft tissues or the brain. Current treatment strategies include surgery, chemotherapy and radiation, without biomarkers to personalize therapy selection or duration and extremely poor patient prognosis. Optionally, immune checkpoint inhibitors (ICI) can be applied in first-line therapy, with PDL1 expression as biomarker for patient selection. Anti-PDL1 therapy has been shown to be highly effective in high mutational burden, adult solid tumors, but evidence for its value for pediatric RT therapy is poorly investigated. Here, we report a comprehensive spatio-temporal profiling of the T cell repertoire for a complete remission (CR) of a pediatric RT patient following adjuvant combinatorial treatment with chemotherapy (doxorubicin, cyclophosphamide, and etoposide) and anti-PDL1 (atezolizumab). Single-cell RNA and TCR sequencing identified an inflamed tumor microenvironment (TME) with extensive clonal expansion of the effector-memory CD8 and T helper/regulatory CD4 compartment. We found notable overlap between tumor-infiltrating and lymphocytes in circulation using ultra-deep single-cell T cell receptor sequencing (OS-T, Omniscope), which allowed to quantify the T cell dynamics throughout ICI treatment (1-year follow-up) and to make informed clinical decisions in real-time. Therapy-induced clonotypes, identified through noise-modeling of the overall deep repertoire distribution, were mostly newly activated CD8 T cells with conserved clonal sizes observed across all time points. We also found a late expansion of tumor-resident CD4 T helper cells that might contribute to form long-lasting anti-tumor immunity. To confirm tumorigenicity, we now validate anti-tumor activity of tumor-resident and therapy-induced clonotypes against the primary tumor in vitro, shortlisting candidate T cells for potential second-line TCR-based therapy. In summary, ultra-deep T cell sequencing of immune cells in circulation enabled the quantification and tracking of therapy-induced T cell clonotypes. Clonal persistence pointed to the induction of long-lasting anti-tumor activity for patient monitoring and identified candidates for personalized TCR-based T cell therapies.</p>
</details>
<details>
<summary>
Lisa Dratva (Wellcome Sanger): <br>
Towards deciphering the T cell receptor-antigen-HLA code with single cell genomics
</summary>
<p>T cells can recognise a plethora of different antigens, presented on HLA molecules, through specific T cell receptors (TCRs) that are unique to individual cells. However, predicting the specificity of a given TCR from its genomic sequence currently presents a major bottleneck for improving our understanding of immune repertoires and developing data-driven approaches for immunotherapies. Recent technological advances in single-cell sequencing have enabled joint profiling of cellular transcriptomes with paired-chain TCR sequences, offering unprecedented insights into disease states and human TCR repertoires. However, interpretation of the data is not straightforward and existing computational tools only focus on individual analysis tasks, such as resolving TCR clones (e.g. Dandelion, MiXCR), sequence similarity-based clustering (e.g. GLIPH, tcrdist3), structural modelling (e.g. AlphaFold), or predicting antigen-HLA binding (e.g. netMHCpan). In order to build a more integrative analysis tool that can facilitate advances in TCR specificity prediction, we have developed Cell2TCR, an open-source package that segments paired-chain TCR sequencing data into functional groups called TCR motifs. Cell2TCR identifies convergent TCR motifs triggered by SARS-CoV-2 infection that are shared across different donors, multiple tissues, and strongly confined to distinct activated T cell states. We experimentally validate the specificity of these TCR motifs and provide compelling statistical evidence for their HLA-restriction. Cell2TCR offers integrated database querying capacities to infer putative antigen specificities for TCR motifs and can inform structural TCR modelling. We showcase the broad applicability of Cell2TCR in various disease contexts, such as infection and cancer. Overall, Cell2TCR represents a powerful and integrative analysis tool for single cell genomics that opens up new avenues for deciphering the TCR-antigen-HLA code.</p>
</details>
<details>
<summary>
Roc Farriol (UCL): <br>
Design of a surface-accessible epitope panel using Brewpitopes to empower early lung cancer detection.
</summary>
<p>Up to 50% of cancer patients are diagnosed at a late stage with tumours that are often unresectable, leading to intensive treatments and a preventable loss of life. Whilst multiple detection screenings have been developed for advanced tumors, many have shown limited sensitivity and specificity for early-stage malignancies. Hence, underscoring an urgent need for novel early detection strategies. Peptide screening to capture antibody signatures in blood has been extensively used in infectious diseases' diagnosis. However, cancer proteins are less foreign to the immune system thus epitope prediction approaches for tumour detection need to prioritise specificity. To this end, we used the SERA discovery platform (Serimmune) to interrogate plasma samples from 60 stage I LUAD patients and analyzed the obtained dataset with IMUNE algorithm to identify an enriched epitope motif shared across the cohort. To generate a customized peptide screening panel and to ensure the surface accessibility of the candidate epitopes, we implemented the Brewpitopes pipeline on the proteins that contain the motif. Brewpitopes works upon protein sequences for linear epitope prediction and crystal structures or Alphafold2 models for conformational epitopes. The pipeline leverages a compendium of state-of-the-art B-cell epitope predictors and a series of bioinformatic tools to map the candidate peptides to extracellular protein regions, to avoid glycosylation sites and to locate them in the 3D surface of the protein to select accessible regions. The target epitope motif mapped to 24 human proteins (251 candidate peptides (11-mers)). The use of Brewpitopes led to an optimized panel comprised of 7 target proteins and 42 extracellular, non-glycosylated and surface-accessible candidates. The resulting panel will be validated in the NIMBLE early lung cancer detection study(>360 patients recruited to date). This study reports for the first time the implementation of Brewpitopes in cancer and displays its capacity to prioritize tumoral antigens for diagnostic purposes. </p>
</details>
</details>
<details open>
<summary>
14:30-15:30 Session 3, Chair: Ursule Demaël
</summary>
<details>
<summary>

<a href="https://scholar.google.co.uk/citations?user=rqbk7ezKqosC&hl=en">Justin Barton</a> (Alchemab Therapeutics / ISMB): <br>
The state of in silico TCR-epitope binding prediction
</summary>
  <p>
  In silico prediction of a T-cell receptor's (TCR) cognate epitope target has been the focus of various methods, yet many fundamental questions regarding the advantages and disadvantages of these approaches remain unanswered. I will present insights from the IMMREP competitions, shedding light on the current state of the art, persisting challenges, and proposing strategies to tackle these challenges. The results underscore the potential of these methods and emphasize the necessity for further independent benchmarking to advance the field.
  </p>
</details>
<details>
<summary>
Olivia Dalby (UCL): <br>
Super-resolution imaging of CAR-T cell immunological synapse
</summary>
<p> The development of T-cells engineered with chimeric antigen receptors (CARs) to recognise tumour antigens on cancer cells constitutes a milestone for T-cell cancer immunotherapies. CARs are synthetic immunoreceptors consisting of an extracellular single-chain antibody fragment and hinge, a transmembrane region, and an intracellular CD3ζ signalling domain that intends to mimic the signal transducing function of the T-cell receptor (TCR) to ultimately kill cancer cells. 
Unlike native T-cell receptors, fewer studies directly assessed the mechanism by which CARs convert extracellular binding events into intracellular signalling. Given that CARs need to integrate into the T-cell signalling network, they are ought to adopt a spatial organisation like that of the TCR. In 2016, images of CARs under total internal reflection fluorescence (TIRF) microscopy suggested that the receptors form immunological synapses analogous to TCR. These results, however, contrast with most recent TIRF experiments that revealed that CAR-T cells form non-classical immune synapses, including disrupted patterns of CARs micro-clusters and absence of the classical actin ring structure. The synapse formation in CAR-T cells is not completely understood and has yet to be visualised with super-resolution fluorescence imaging. Furthermore, there is a noticeably lack of information of the factors that contribute to their structure.
Here, we quantify the spatial distribution of CARs and pCD3ζ on the surface of resting and activated primary human T-cells using DNA-PAINT, a single-molecule localisation microscopy technique capable of single protein resolution. Actin remodelling is also evaluated by imaging activated CAR-T cells on a spinning disk super-resolution by optical pixel reassignment (SoRa) microscope. Notably, by using glass supported lipid bilayer as a sample format that can mimic the T-cell / target interface, our studies focus on the effects of antigen density on the CARs immunological synapse structure. Clinically, this is relevant as cancer cells commonly adapt to reduce expression of ligands, subsequently making the treatment ineffective. Among the CARs used in clinical trials, the most extensively investigated one is anti-CD19 CAR, targeting CD19 molecules expressed in B-cell lymphomas or leukaemia. Still, the proportion of patients who respond to CD19-targeted therapy remains modest (40%) and up to a third see their disease return within a year. Using anti-CD19 CARs as a model system, our imaging and data analysis reveal that only CAR-T cells exposed to high ligand densities (> 350 CD19 molecules/µm2) present a disrupted immunological synapse formation (i.e., distinct CARs micro-clusters formation and an actin network that is not fully cleared).   
The combination of these imaging studies begins to uncover the picture of a disrupted CAR T cell IS formed on activation. These results mark the first single-molecule imaging study into the activation of CAR T cells. Subsequently, this work begins to build the understanding of CAR-T cell activation mechanisms and benefit the future design of new CAR-T treatments against aggressive cancers.  </p>
</details>
<details>
<summary>
Jose Cabezas-Caballero (Oxford): <br>
Genetically engineering T cells to reduce the risk of autoimmune cross-reactivities in T cell therapies
</summary>
<p> T cell receptor-engineered T cell (TCR-T) therapies are limited by the potential risk of cross-reactivity against healthy tissues. For example, T cells engineered with the MAGE-A3 specific a3a TCR caused lethal autoimmune toxicities in patients due to cross-reactivity to a lower affinity peptide derived from the muscle protein titin. Therefore, there is an urgent need to identify methods that reduce T cell activation to lower-affinity self-peptides, whilst maintaining a potent response to higher-affinity target peptides. We have selected the well-studied NY-ESO-1 specific c259 TCR as a model system to investigate ligand discrimination. Firstly, we have measured the affinity of the c259 TCR to a panel of 8 peptides by Surface Plasmon Resonance. Secondly, we performed a targeted CRISPR-Cas9 knock-out screen in human primary T cells to identify genetic modifications that can modulate ligand discrimination. Finally, we measured the affinity of the c259 TCR to a positional scanning library (171 peptides) and tested the impact of our genetic modifications on T cell cross-reactivity. Using our 8 peptide NY-ESO-1 panel, we have identified genetic modifications that can selectively reduce T cell activation against lower affinity peptides, whilst maintaining a potent response to higher affinity peptides. These genetic modifications can also abolish a3a TCR cross-reactivity to the lower affinity titin peptide, without reducing activation against MAGE-A3. Additionally, we have demonstrated that our genetically engineered T cells have reduced cross-reactivity using the 171 peptide NY-ESO-1 positional scanning library.Our findings suggest that T cells can be engineered to exhibit different degrees of ligand discrimination. This approach could be applied to other clinically relevant TCRs for the development of T cell therapies with reduced risk of lethal autoimmune toxicities. Furthermore, the combination of our c259 TCR positional scanning peptide library affinity and activation data could be used to train machine learning models to better predict TCR-peptide specificity and cross-reactivity. I will be describing the current state of the art in both computational and experimental technologies aiming to reconstruct a map between T cells and their cognate antigens. I will highlight the remaining challenges and put forward a few ideas on how to tackle the challenges. </p>
</details>
</details>
<details open>
<summary>
15:45-16:15 Closing Session
</summary>
<details>
<summary>
<a href="https://kabirhusain.github.io/">Kabir Husain</a> (UCL): <br>
The noise is the signal
</summary>
  <p>
  Eighty years ago, Luria and Delbruck had a rather clever idea: simply by counting bacterial colonies on a plate, they learned something about the nature of evolution and even measured the mutation rate -- a full decade before the structure of DNA was determined. However, their particular statistical insight has not been brought into the modern age. I will present a high-throughput riff on their work, and discuss applications in the context of ongoing work on proofreading DNA polymerases, modifier loci, and protein evolvability. Overall, I hope to suggest a phenomenological, statistical approach to study the ebb and flow of genetic variation in a population.
  </p>
</details>
</details>
- 16:15-17:15 Posters and drinks  

Toggle to see abstracts.

## Talks and Posters

We highly encourage you to contribute to the London Q-Immuno day - works in progress and contributions from early career researchers are particularly welcome!

There are three possible formats:

(1) a **short talk** during one of the main sessions  
(2) a **whiteboard talk**: a talk without slides where you will explain your project interactively to a small audience with the support of a marker and a whiteboard  
(3) a **poster** 

If you would like to be considered for any of the three, please indicate so when [registering](https://lu.ma/9rvihe65). If you want to be considered for a short or whiteboard talk  the deadline to submit a title and abstract is the **28th of February**.


## Organisers

[Yuta Nagano](https://www.yutanagano.com/), [Ursule Demaël](https://twitter.com/UDemael), [James Henderson](https://qimmuno.com/team/) and [Andreas Tiffeau-Mayer](https://qimmuno.com/).

The LDN Q-Immuno Day is made possible by a [USyd-UCL Global Engagement award](https://www.ucl.ac.uk/global/ucl-global-engagement) and UCL's [Institute of Immunity and Transplantation](https://www.ucl.ac.uk/immunity-transplantation/ucl-institute-immunity-and-transplantation) and [Institute for the Physics of Living Systems](https://www.ucl.ac.uk/physics-living-systems/institute-physics-living-systems).  

It is part of a series of events organised by the informal [London Quantitative Immunology Network](https://qimmuno.com/ldnqimmuno/). Please join our mailing list!
