---
layout: conf
title: LDNQImmuno
subtitle: March 9th
order: 5
---

<div class="titlebox">
  <h1>
  London Quantitative Immunology Day
  </h1>
  <div class="box">
  <span style="text-align:left">
    Thursday March 9th 2023 <br /> 
    Pears Building, UCL IIT <br />
    Pond Street, London NW32PP <br />
    Tube station: Belsize Park <br />
  </span>
  </div>
  <p style="margin-top:0em;padding:0.5rem">
    A community day for researchers in the quantitative life sciences from across London. <br />
  </p>
  <p style="margin-top:0em;padding-top:0em;padding-bottom:1em;font-size:150%">
   <a href="https://forms.office.com/Pages/ResponsePage.aspx?id=_oivH5ipW0yTySEKEdmlwsefyMHuSa9KkArfAHERWzNUQ0tOMlJIMElFUEVLNTM0T0RHMk9XSDZaSy4u">&rarr; Register now &larr;</a>
  </p>
</div>

<img style="max-width:55%;margin:0.2rem;padding-right:0;margin-right:0" src="../images/ldnday23.png" align="right">
We aim to bring together researchers with an interest in quantitative immunology to create an **opportunity for sharing knowledge and social exchange**. It is going to be a day of conviviality and scientific enthusiasm with a dynamic and informal atmosphere. Talks from invited speakers will be interleaved with short presentations by young investigators (contributions welcome!). The schedule includes ample breaks for discussions and, for those interested, we propose to conclude the day in a local pub.

Immunology is being transformed through a multitude of quantitative approaches. Correspondingly, we expect the day to be of interest to researchers from diverse backgrounds from immunology, evolution, and cell biology, to systems biology, bioinformatics, and the physics of living systems. **No matter your background, you are welcome to join us!**

There is no registration fee, but we encourage prior [registration](https://forms.office.com/Pages/ResponsePage.aspx?id=_oivH5ipW0yTySEKEdmlwsefyMHuSa9KkArfAHERWzNUQ0tOMlJIMElFUEVLNTM0T0RHMk9XSDZaSy4u) to help us gauge attendance.

## We want YOU!

We highly encourage you to contribute to the London QImmuno day - works in progress and contributions from early career researchers are particularly welcome!

There are three possible formats:

(1) a **short talk** during one of the main sessions  
(2) a **whiteboard talk**: a talk without slides where you will explain your project interactively to a small audience with the support of a marker and a whiteboard  
(3) a **poster** 

If you would like to be considered for any of the three, please indicate so when [registering](https://forms.office.com/Pages/ResponsePage.aspx?id=_oivH5ipW0yTySEKEdmlwsefyMHuSa9KkArfAHERWzNUQ0tOMlJIMElFUEVLNTM0T0RHMk9XSDZaSy4u). If you want to be considered for a short or whiteboard talks, please reply by February 22nd.

## Schedule

- 09:30-10:00 Meet and Greet  
<details open>
<summary>
10:00-11:00 Session 1, Chair: Andreas Mayer
</summary>
<details>
<summary>
<a href="https://www.imperial.ac.uk/people/b.bravi21">Barbara Bravi</a> (Imperial): <br> 
Machine learning models to model antigen immunogenicity and T-cell recognition
</summary>
<p>Antigen immunogenicity and the specificity of binding of T-cell receptors to antigens are key properties underlying effective immune responses. Identifying immunogenic antigens, as well as antigen-specific T-cell receptors, is therefore crucial to vaccine and cancer immunotherapy design. In this talk, I will discuss a set of flexible and easily interpretable methods that we have recently developed based on the machine learning scheme of Restricted Boltzmann Machines (RBM). Such scheme allowed us first to build models of antigen presentation by the human leukocyte antigen class I proteins and antigen immunogenicity, which can be used to reconstruct the underlying molecular motifs and as predictors of viral epitopes and cancer neoantigens. I will next introduce RBM-based models of the complementary process of recognition by T cells of presented antigens, which are able to discriminate responses specific to different epitopes and to detect signatures of response at the T-cell repertoire level.</p>
</details>
<details>
<summary>
Megan Joseph (UCL): <br>
qPAINT analysis of podoplanin clustering in fibroblastic reticular cells uncovers CD44 function
</summary>
<p> The lymph node is a highly organised structure that contains multiple immune cell types. Organisation of the lymph node is critical to healthy immune function. Fibroblastic reticular cells (FRCs) are structural cells that establish lymph node architecture. During immune invasion a plethora of immune cell populations flood into the lymph node. To cope with rapid enlargement FRC networks stretch allowing the lymph node to expand. Upon initial immune challenge, dendritic cells (DCs) enter the lymph node and interact with FRCs via C-type lectin-like receptor 2 (CLEC-2).  CLEC-2 binds to podoplanin (PDPN) found on the surface of FRCs. Research has shown that interaction of CLEC-2 and PDPN inhibits interaction with actin pathways that promote FRC contraction. CD44, another FRC transmembrane protein, has been shown to interact with PDPN upon CLEC-2 binding. Yet, its function is not yet fully elucidated. Here, we use a quantitative single molecule super-resolution technique named qPAINT to shed light upon the relationship between PDPN clustering and CD44 in the membrane of FRCs upon CLEC-2 interaction. Specifically, via qPAINT, we quantify PDPN clustering in CD44 knock out and wild type FRC’s, both in resting and CLEC-2 treated conditions. Our results indicate that CLEC-2 interaction leads to the formation of large clusters of PDPN (i.e., more than 6 proteins per cluster) in a CD44 dependent manner. This suggests that CD44 stabilizes large pools of PDPN at the membrane of FRCs upon CLEC-2 interaction, possibly to aid inhibition of contraction promoting actin polymerization.</p>
</details>
<details>
<summary>
Nicholas McGranahan (UCL): <br>
Lung cancer immune escape through HLA disruption
</summary>
<p>Disruption of the human leukocyte antigen (HLA) molecules has important implications for immune evasion and tumour evolution. However, although genomic loss of HLA is frequent in non-small cell lung cancer (NSCLC), the extent and importance of transcriptomic disruption to HLA presentation, including transcript repression and alternative splicing, remains unclear. Here we look beyond the genome to explore HLA disruption in cancer evolution. </p>
</details>
</details>
- 11:00-11:30 Coffee break  
<details open>
<summary>
11:30-12:30 Session 2, Chair: Leo Swadling
</summary>
<details>
<summary>
<a href="https://dushek.path.ox.ac.uk/">Omer Dushek</a> (Oxford): <br>
The discriminatory power of the T cell receptor
</summary>
 <p>T cells use their T cell receptors (TCRs) to discriminate between lower-affinity self and higher-affinity non-self peptides presented on major histocompatibility complex (pMHC) antigens. Although the discriminatory power of the TCR is widely believed to be near-perfect, technical difficulties have hampered efforts to precisely quantify it. Using an accurate method for measuring very low TCR/pMHC affinities, we quantify the discriminatory power of the TCR. We find that TCR discrimination, although enhanced compared with conventional cell-surface receptors is imperfect: primary human T cells can respond to pMHC with affinities as low as KD ∼ 1 mM. The kinetic proofreading mechanism fit our data, providing the first estimates of both the time delay (2.8 s) and number of biochemical steps (2.67) that are consistent with the extraordinary sensitivity of antigen recognition. We next identify factors that can control antigen discrimination, including the accessory receptors CD2 and LFA-1. Taken together, our findings explain why self pMHC frequently induce autoimmune diseases and anti-tumour responses, and suggest ways to modify TCR discrimination for improved therapies.</p>
</details>
<details>
<summary>
Luis Zapata (ICR): <br>
Cancer evolution driven by immunoediting and immune escape
</summary>
 <p>International cancer consortiums, fuelled by the advent of sequencing technologies, have paved the way for a better understanding of human disease, enabling the scientific community to explore the cancer genome. While most studies have focused on detecting the landscape of genes under positive selection, few have investigated how neutral drift and negative selection interact with the cancer genome. Here, I used the ratio of nonsynonymous to synonymous substitutions (dN/dS) to detect immune selection in more than 10000 tumor genomes.  However, the extent of negative selection in cancer remains highly controversial and is difficult to quantify due to limited number of mutations in patients. To address this challenge, methods combine multiple patients into a single cohort, mixing opposing evolutionary trajectories, and masking negative selection.  I explored this hypothesis and demonstrated that immune selection triggers two possible evolutionary scenarios: cells become either antigen-free or immune-escaped. We showed that by mixing antigen-free with immune-escaped tumors, the signal of immune-mediated negative selection is lost. In addition, we demonstrated that low dN/dS (antigen free) patients have the worst prognosis upon checkpoint inhibitor, suggesting a novel mechanism of primary resistance to treatment. </p>
</details>
<details>
<summary>
Jodie Chandler (UCL): <br>
Investigating the proliferative history of flu-specific memory T cell subsets
</summary>
 <p>By utilising novel Ki67 fate reporting mice in a lung model of influenza we accurately label proliferating cells at desired times throughout both the effector, memory and recall immune response. Harnessing the power of quantitative immunological approaches, we are then able to calculate the division history of various memory T cell subsets with an aim to shed light on the origin, persistence, destiny, and overall dynamics of T cell memory. </p>
</details>
</details>
- 12:30-13:45 Lunch break
<details open>
<summary>
13:45-14:30 Whiteboard session  
</summary>
<details>
<summary>
Peter Thomas (UCL): <br>
B cell repertoire diversity analysis using masked language modelling
</summary>
</details>
<details>
<summary>
Luisa Chocarro (Navarrabiomed/UCL): <br>
PD-1/LAG-3 dysfunctionality signatures in human cancers
</summary>
</details>
</details>
<details open>
<summary>
14:30-15:30 Session 3, Chair: Jennifer Cowan
</summary>
<details>
<summary>
<a href="https://www.hoogenboom-lab.com/">Bart Hoogenboom</a> (UCL): <br>
Physical membrane properties protect cytotoxic T cells from suicide
</summary>
  <p>To eliminate virus-infected and tumour cells, cytotoxic T cells form a synapse with their target, in which they release pro-apoptotic granzymes as well as pore forming proteins called perforin. Perforin punches holes in the target membrane to facilitate cell entry for the granzymes, which next initiate a signalling cascade leading to programmed cell death (apoptosis). It has long been a mystery what protected the T cells from being targeted by the proteins they secrete to kill their targets. I will report on recent studies that have identified such self-protection mechanisms based on the physical properties of the T cell membrane. Notably, enhanced lipid order and packing reduce perforin binding, whereas externalisation of negatively charged lipids results in local deactivation of perforin. This raises the yet unanswered question if similar membrane-based protection could be used by cancer cells to evade immune killing.
  </p>
</details>
<details>
<summary>
Heather Jackson (Imperial): <br>
Using host whole blood transcriptomics to improve diagnosis of multisystem inflammatory syndrome in children (MIS-C)
</summary>
<p> Multisystem inflammatory syndrome in children (MIS-C) occurs 2-6 weeks following SARS-CoV-2 infection in some children. Diagnosis of MIS-C is challenging due to its overlapping clinical presentation with other paediatric infectious and inflammatory diseases. We have found a combination of 5 host RNA transcripts that, when combined, can distinguish MIS-C from other paediatric infectious and inflammatory diseases with high accuracy. </p>
</details>
<details>
<summary>
Hashem Koohy (Oxford): <br>
Can we predict T cell specificity with digital biology and machine learning?
</summary>
<p> I will be describing the current state of the art in both computational and experimental technologies aiming to reconstruct a map between T cells and their cognate antigens. I will highlight the remaining challenges and put forward a few ideas on how to tackle the challenges. </p>
</details>
</details>
- 15:30-17:00 Poster and drinks  

Toggle to see abstracts.

## Organizers

[Martina Milighetti](https://twitter.com/martina_milig), [Yuta Nagano](https://www.yutanagano.com/), and [Andreas Mayer](https://qimmuno.com/).

The LDN QImmuno Day is made possible by UCL's [Institute of Immunity and Transplantation](https://www.ucl.ac.uk/immunity-transplantation/ucl-institute-immunity-and-transplantation) and [Institute for the Physics of Living Systems](https://www.ucl.ac.uk/physics-living-systems/institute-physics-living-systems).
