---
layout: conf
title: LDNQImmuno
subtitle: April 10th
order: 5
---

<div class="titlebox">
  <h1>
  3rd London Quantitative Immunology Day
  </h1>
  <div class="box">
  <span style="text-align:left">
    Thursday April 10th 2025 <br /> 
    Pears Building, UCL IIT <br />
    Pond Street, London NW32PP <br />
    Tube station: Belsize Park <br />
  </span>
  </div>
  <p style="margin-top:0em;padding:0.5rem">
    A community day for researchers in the quantitative life sciences from across London. <br />
  </p>
  <p style="margin-top:0em;padding-top:0em;padding-bottom:1em;font-size:150%">


<script id="luma-checkout" src="https://embed.lu.ma/checkout-button.js"></script>

<a
  href="https://lu.ma/event/evt-tYayjvtZowF9VXU"
  class="luma-checkout--button"
  data-luma-action="checkout"
  data-luma-event-id="evt-tYayjvtZowF9VXU"
>
  Register now
</a>

  </p>
</div>

<img style="max-width:55%;margin:0.2rem;padding-right:0;margin-right:0" src="../images/ldnday25.jpg" align="right">
We aim to bring together researchers with an interest in quantitative immunology to create an **opportunity for sharing knowledge and social exchange**. The London Q-Immuno day will be a day of conviviality and scientific enthusiasm with a dynamic and informal atmosphere. Talks from invited speakers will be interleaved with short presentations by young investigators (contributions welcome!). The schedule includes ample breaks for discussions and, for those interested, we propose to conclude the day in a local pub.

Immunology is being transformed by the application of a multitude of quantitative methods. We want to foster discussions between researchers from diverse backgrounds: immunology, evolution, computational biology, evolution, systems biology, bio-informatics, mathematics and the physics of living systems. **No matter your background, you are welcome to join us!**

There is no registration fee, but we encourage prior [registration](https://lu.ma/sg42wn3p) to help us gauge attendance.

## Schedule

- 09:30-10:00 Meet and Greet
<details open>
<summary>
10:00-11:00 Session 1, Chair: Andreas Tiffeau-Mayer
</summary>
<details>
<summary>
<a href="https://trr186.uni-heidelberg.de/en/ThomasHofer">Thomas Höfer</a> (Heidelberg): <br>
A lineage tree for tissue macrophages
</summary>
<p>Elucidating the clonal dynamics of immune cell populations, and the underlying laws of cell behavior, is a key challenge of cellular immunology. I will briefly review model-based inference approaches we have developed for diverse types of experimental data. I will then focus on tissue-resident macrophages. Their ontogeny remains controversial, with many current models invoking multiple distinct waves of macrophage development. In joint work with the Rodewald lab, we developed a holistic approach, barcoding the entire mouse embryo at successive developmental stages and mathematically inferring macrophage development from gastrulation to adult organs. Our data reveal a continuous lineage tree of tissue macrophages. This tree originates from a pan-hematopoietic progenitor at embryonic day (E)6.5 and branches, via oligolineage progenitors (E7.5-E9.5), into organ-specific macrophage lineages by E10.5. Organ-specific progenitors form local macrophage colonies that persist into adulthood. This model makes quantitative and qualitative predictions that we have begun to test.</p>
</details>
<details>
<summary>
Ilinca Patrascan (Imperial): <br>
Multimodal tracking of T cell lineage choice and differentiation
</summary>
<p> T cell fate decision can be described as a continuum of transcriptional states that are not always accurately captured by cell surface markers or steady-state transcript levels. RNA fluorescence in situ hybridisation (smRNA-FISH) allows the detection of nascent transcripts and the enumeration of mature transcripts. Here we harness the idea that proteins, mature transcripts and primary transcripts turn over at different rates - slow for proteins, intermediate for mature (spliced) mRNAs and rapid for nascent pre-mRNA. Tracking gene transcription at a single molecule level provides a powerful approach to view a cell's present state in the context of its developmental trajectory. During T cell differentiation, bipotent progenitors commit to either the CD4+ helper or the CD8+ cytotoxic lineage. Pre-selection cells exhibit stable ratios of protein, mRNA and primary transcripts. Entry into the selection process triggers a ratio change where primary transcripts form the leading edge, mature RNA reflects recent and proteins more distant gene expression. Linear differentiation maintains coherent ratios that reveal lineage direction and speed, while incoherent ratios signal a change in differentiation trajectory (non-linear). This distinction thus allows us to test current models of T cell lineage choice, and to characterise the molecular basis of CD4 versus CD8 lineage differentiation.</p>
</details>
<details>
<summary>
Noah Grodzinski (Cambridge): <br>
Wetting of active Brownian particles
</summary>
<p>I will present my current work on wetting of Active Brownian Particles (ABPs), in collaboration with Mike Cates and Robert Jack in the Soft Matter Group in Cambridge. I will give a brief review of the central concepts of active matter, presented in a mathematically accessible manner for a general audience. I will then briefly review our recent work on the topic, on wetting of ABPs on a permeable barrier. Finally I will discuss how these concepts can be applied to modelling biological systems. </p>
</details>
</details>
- 11:00-11:30 Coffee break
<details open>
<summary>
11:30-12:30 Session 2, Chair: Ursule Demaël
</summary>
<details>
<summary>
<a href="https://www-sormanni.ch.cam.ac.uk/">Pietro Sormanni</a> (Cambridge): <br>
Third generation approaches of antibody discovery and engineering
</summary>
 <p>Antibodies are indispensable in research, diagnostics, and as therapeutics. Despite significant advances in antibody discovery and engineering technologies, challenges remain, particularly in the targeting of predetermined epitopes and the simultaneous optimisation of multiple biophysical traits. Traditional screening methods can be labor-intensive and struggle to navigate the complex trade-offs between properties such as affinity, specificity, stability, and solubility. It is increasingly possible to complement well-established in vivo (first generation) and in vitro (second generation) methods of antibody discovery with in silico (third generation) approaches, which offer speed, cost-effectiveness, and resource efficiency. In this presentation, I will explore emerging computational methods of antibody design, which enable precise targeting of specific epitopes, accurate prediction of nativeness, nanobody humanisation, and the optimisation of developability potential through the simultaneous enhancement of multiple biophysical properties. These approaches can streamline antibody development, enabling to address new questions and paving the way for rapid advancements in therapeutic and diagnostic applications.
</p>
</details>
<details>
<summary>
Constantin Ahlmann-Eltze (UCL): <br>
A pan-tissue atlas of T cells in early cancer development
</summary>
<p> Early detection of tumors prolongs survival across many cancer types. Despite improving screening methods and treatment options, there is still a need for improved biomarkers and drug targets that can intercept tumor development. Here, we present a pan-tissue atlas of the tumor microenvironment, where we compare the cell type composition of healthy, pre-malignant, and tumor tissue. We assembled publicly available data for ten tissues from 21 datasets and 331 donors. To provide consistent cell type labels and robust analysis, we developed a novel tool called treelabel that stores cell type annotations for each cell at various resolutions. This allows us to choose the optimal resolution to detect abundance changes that are reproducible across tissue types and annotation methods. We find that already in the pre-malignant stage, the microenvironment is characterized by an immune-suppressive phenotype. Furthermore, we show that the composition of T cell subtypes can be used as a biomarker to distinguish healthy from pre-malignant and cancerous tissue.
 </p>
</details>
<details>
<summary>
Alexis Farman (UCL): <br>
Enhancing immunotherapies: Insights from the mathematical modelling of a microfluidic device.
</summary>
 <p>A pivotal aspect of developing effective immunotherapies for solid tumours is the robust testing of product efficacy inside in vitro platforms.Collaborating with an experimental team that developed a novel microfluidic device at Children’s National Hospital (CNH), we developed a mathematical model to investigate immune cell migration and cytotoxicity within the device.
Specifically, we study Chimeric Antigen Receptor (CAR) T-cell migration inside the channels, treating the cell as a moving boundary driven by a chemoattractant concentration gradient. The chemoattractant concentration is governed by two partial differential equations (PDEs) that incorporate key geometric elements of the device. We examine the motion of the cell as a function of its occlusion of the channel and find that certain cell shapes allow for multiple cells to travel inside the channel simultaneously. Additionally, we identify parameter regimes under which cells clog the channel, impairing their movement. All our findings are validated against experimental data provided by CNH.
We integrate our model results into a broader model of the device, which also examines the cytotoxicity of CAR T-cells. This provides a tool for distinguishing experimental artefacts from genuine CAR T-cell behaviour. This collaboration enabled the team at Children’s National Hospital to refine experimental conditions and uncover mechanisms enhancing CAR T-cell efficacy. </p>
</details>
</details>
- 12:30-13:45 Lunch break
<details open>
<summary>
13:45-14:30 Whiteboard session
</summary>
<details>
<summary>
Michael Schneider (ETH): <br>
Structure-Enhanced Protein Large Language Model Representations
</summary>
</details>
<details>
<summary>
Alastar Phelan (Imperial) <br>
Optimising the signal in cell cycle analysis by dual pulse nucleoside labelling experiments
</summary>
</details>
</details>
<details open>
<summary>
14:30-15:30 Session 3, Chair: Leo Swadling
</summary>
<details>
<summary>
Aubin Ramon (Cambridge): <br>
Prediction of protein biophysical traits from limited data: a case study on nanobody thermostability through NanoMelt
</summary>
<p> Predicting protein biophysical traits is challenging due to limited and heterogeneous data. We present a comprehensive study on protein fitness prediction from limited data, leveraging pre-trained embeddings and ensemble learning. Using this framework, we introduce NanoMelt, a nanobody thermostability predictor trained on 640 melting temperature measurements, including 129 new ones. NanoMelt achieves state-of-the-art accuracy and streamlines nanobody development by guiding the selection of highly stable candidates. </p>
</details>
<details>
<summary>
Pebs Edwards (ICR): <br>
T cell clonal expansion in peripheral blood of Lynch syndrome patients upon detection of mismatch repair-deficient tumours and polyps
</summary>
 <p>The T cell receptor (TCR) repertoire reflects adaptive immune responses and plays a critical role in immune surveillance and tumour recognition. Lynch syndrome (LS) is an inherited cancer predisposition caused by germline pathogenic variants in mismatch repair (MMR) genes. Somatic loss of the second healthy allele leads to MMR deficiency (dMMR), resulting in microsatellite instability and an increased risk of developing tumours, including colorectal cancer (CRC). However, the earliest events of CRC initiation in LS remain poorly understood. Previous studies from our lab have shown that dMMR crypt foci are extremely rare in non-neoplastic colon tissue and we hypothesise that although dMMR cells arise frequently they are efficiently cleared by the immune system. This study aims to characterise the TCR repertoire in the peripheral blood of LS patients to explore immune responses to dMMR cells.
Methods:
Blood-derived TCR repertoires were compared across 181 patients divided into four groups, healthy individuals without LS (n = 55), healthy individuals with LS (n = 95), dMMR cancer patients without LS (n = 19), dMMR cancer patients with LS (n = 12). Quantitative RNA-based TCR sequencing was performed and bioinformatic analyses were conducted using the R package Immunarch.
Results:
We examined unique clonotypes, diversity (by Inverse Simpson) and frequency of hyperexpanded clonotypes in peripheral blood TCR repertoires. LS patients exhibited significantly fewer unique clonotypes than non-LS patients (mean = 3781 vs 4061, p = 0.0015). LS patients also had significantly lower diversity (mean = 1270) compared to non-LS (mean = 2123) (p = 8.6e-06). Comparing LS and non-LS patients revealed a trend toward more large (p = 0.0037) and hyperexpanded (p = 0.0624) clonotypes in LS blood suggestive of clonal expansion.
In patients with cancer, the number of unique clonotypes in peripheral blood was significantly reduced when colonic polyps or cancer were present: numbers of unique clonotypes from normal (mean = 4070), to polyp (mean = 3713), to cancer (mean = 3400; p = 3.5e-09). TCR repertoire diversity (by Inverse Simpson) also showed significant differences based on colonoscopy findings (p = 1.2e-05), with reduced diversity in cancer (mean = 953), followed by polyp (mean = 1033) and normal tissue (mean = 1937). A non-significant trend towards increased large and hyperexpanded clonotypes was found in patients with cancer, and to a lesser extent in patients with polyps.
Conclusions:
These findings suggest that T cell clonal expansion in blood can be driven by the presence of colonic dMMR tumours, or to a lesser extent with colonic polyps. Further experimental and bioinformatic analyses are needed to match frameshift peptides with specific TCRs; this would provide insights into antigen-driven immune responses in LS-associated CRC and polyps.
</p>
</details>
<details>
<summary>
<a href="https://sites.google.com/view/xiaowen-chen">Xiaowen Chen</a> (LPENS): <br>
Inferring resource competitions in microbial communities from time series
</summary>
  <p>The competition for resources is a defining feature of microbial communities. In many contexts, from soils to host-associated communities, highly diverse microbes are organized into metabolic groups or guilds with similar resource preferences. The resource preferences of individual taxa that give rise to these guilds are critical for understanding fluxes of resources through the community and the structure of diversity in the system. However, inferring the metabolic capabilities of individual taxa, and their competition with other taxa, within a community is challenging and unresolved. In this talk, I will address this gap in knowledge by leveraging dynamic measurements of abundances in communities [1]. I will show that while simple correlations are often misleading in predicting resource competition, spectral methods such as the cross-power spectral density (CPSD) and coherence that account for time-delayed effects are superior metrics for inferring the structure of resource competition in communities. I will first demonstrate this fact on synthetic data generated from consumer-resource models with time-dependent resource availability, where taxa are organized into groups or guilds with similar resource preferences. Then, by applying spectral methods to oceanic plankton time-series data [2], we will demonstrate that these methods detect interaction structures among species with similar genomic sequences. Our results indicate that analyzing temporal data across multiple timescales can reveal the underlying structure of resource competition within communities. Finally, If time permits, I will also show progress towards understanding the impact of community diversity (e.g guild-size variability and species
uneveness) on the guild-structure reconstruction success.
  </p>
</details>
</details>
- 15:30-17:00 Poster and drinks
- 17:00-late Local Pub

Toggle to see abstracts.

## Talks and Posters

We highly encourage you to contribute to the London Q-Immuno day - works in progress and contributions from early career researchers are particularly welcome!

There are three possible formats:

(1) a **short talk** during one of the main sessions  
(2) a **whiteboard talk**: a talk without slides where you will explain your project interactively to a small audience with the support of a marker and a whiteboard  
(3) a **poster** 

If you would like to be considered for any of the three, please indicate so when [registering](https://lu.ma/sg42wn3p). If you want to be considered for a short or whiteboard talk  the deadline to submit a title and abstract is the **27th of March**.


## Organisers

Matthew Cowley, Ursule Demaël, James Henderson and Andreas Tiffeau-Mayer.

The LDN Q-Immuno Day is made possible by UCL's [Institute of Immunity and Transplantation](https://www.ucl.ac.uk/immunity-transplantation/ucl-institute-immunity-and-transplantation) and [Institute for the Physics of Living Systems](https://www.ucl.ac.uk/physics-living-systems/institute-physics-living-systems).  

It is part of a series of events organised by the informal [London Quantitative Immunology Network](https://qimmuno.com/ldnqimmuno/). Please join our mailing list!
